bcda/bcdacli/cli.go

```diff
@@ -433,6 +435,67 @@ func setUpApp() *cli.App {
 				return nil
 			},
 		},
+		{
+			Name:     "generate-synthetic-alr-data",
+			Category: "Data import",
+			Usage:    "Generate and ingest synthetic ALR data associated with a particular ACO",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "cms-id",
+					Usage:       "CMS ID of ACO",
+					Destination: &acoCMSID,
+				},
+				cli.StringFlag{
+					Name:        "alr-template-file",
+					Usage:       "File path for ALR template file",
+					Destination: &alrFile,
+					Value:       "./alr/gen/testdata/PY21ALRTemplatePrelimProspTable1.csv",
```

Comment:
Do you see this location changing?

Reply:
Good call - i just removed it from the CLI